cmd/cluster/core/create.go

```diff
@@ -70,6 +70,7 @@ type CreateOptions struct {
 	Wait                             bool
 	Timeout                          time.Duration
 	Log                              logr.Logger
+	FromLocal                        bool
```

Comment:
Can we change the name of the flag to something more specific like `SkipAPIBudgetVerification` ? Whether we send metrics or not from the test is not controlled by this flag. All it controls is whether we verify the budget.